packages/ui/src/kit/VRadio/VRadio.theme-default.scss

```diff
@@ -35,8 +39,8 @@
   content: '';
   display: block;
   position: absolute;
-  top: 5px;
-  left: 5px;
+  top: 4px;
+  left: 4px;
```

Comment:
also add `box-sizing: border-box` because circle looks badly
![image](https://user-images.githubusercontent.com/756731/78346543-d3b40500-75a7-11ea-99d3-226aa88b5597.png)